### PR TITLE
LTD-537 Update name of GovPermission

### PR DESCRIPTION
**Pull request by r4vi**
change enum value from 'Confirm own advice' to 'Manage own advice'
to better represent the intent of this permission

This was not as simple as updating the enum as the values are loaded into the database during the initial seed operation and are not updated again so a special data migration is needed to make this change manifest in the UI

### CODE REVIEW

File: api/users/migrations/0015_auto_20210212_1620.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    """
+    Update Permission objects in DB if their display value has changed
+    """
+    Permission = apps.get_model("users", "Permission")
+    for gp in GovPermissions:
+        try:
+            # if any permission exist in the db with a name that doesn't match
+            # the current value of the enum then update them
+            # otherwise continue
+            perm = Permission.objects.get(~Q(name=gp.value), id=gp.name, type=UserType.INTERNAL)
```
Comment (saruniitr) [inline api/users/migrations/0015_auto_20210212_1620.py:21]: `id=gp.id`?

Comment (r4vi) [reply api/users/migrations/0015_auto_20210212_1620.py:21]: confusingly the GovPermission is an enum with `name, value` attributes, which are converted to Permissions objects as `Permission(id=gp.name, name=gp.value)`

File: api/users/migrations/0015_auto_20210212_1620.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            old = perm.name
+            perm.name = gp.value
+            perm.save()
+            print(f"Updated {perm.name} with display value: {old} -> {perm.name}")
```
[the remark this answers is not in the document]
Comment (r4vi) [reply api/users/migrations/0015_auto_20210212_1620.py:25]: I think it's useful to see if the migration does perform any action but yes happy to take out

[the remark this answers is not in the document]
Comment (saruniitr) [reply api/users/migrations/0015_auto_20210212_1620.py]: No, please leave it if it is intentional

File: api/users/migrations/0015_auto_20210212_1620.py (hunk 1 above)
Comment (saruniitr) [reply api/users/migrations/0015_auto_20210212_1620.py:21]: :+1:

**Review by saruniitr — APPROVED** [review]
May be please add a test to see if no permissions exist with the name "Confirm own advice".

File: api/users/migrations/0015_auto_20210212_1620.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (r4vi) [reply api/users/migrations/0015_auto_20210212_1620.py]: I think you're right though, random prints in migrations is bad. I've removed it

**Comment by r4vi** [comment]
> May be please add a test to see if no permissions exist with the name "Confirm own advice".

I don't think a test will prove much as when the tests run it will seed using current values so this migration will just silently pass through.

**Pull request merged**